resolver/src/system_conf/windows.rs

```diff
@@ -0,0 +1,55 @@
+use std::io;
```

Comment:
I like the separate module.

I'm surprised there is no warning about module docs missing, i.e. `//!` at the top of this file...

Reply:
Probably because it's not a public module. I'll add something (and the copyright block that I forgot)